src/pages/Components/Section/ChangePW.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from 'react';
+import { ReactComponent as Prev } from 'assets/svg/prev-icon.svg';
+import warning from 'assets/svg/warning.svg';
+import style from './SearchId.module.scss';
+import ModalForPW from './ModalForPW';
+
+export default function ChangePW() {
+  const pattern = /^.*(?=^.{2,16}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%^&+=]).*$/;
+  const [pw, setPw] = useState<string>('');
+  const [pwch, setPwch] = useState<string>('');
+  const [Valid, setValid] = useState<string>('initial');
+  const [isEmpty, setIsEmpty] = useState(true);
+  const [isSuccess, setIsSuccess] = useState(false);
+  const checkEmpty = () => {
+    if (pw !== '' && pwch !== '') setIsEmpty(false);
+    else if (pw === '' || pwch === '') setIsEmpty(true);
+  };
```

Comment:
개발하시는 도메인을 몰라 확실하진 않지만 아래로 충분하지 않나요?

```suggestion
  const checkEmpty = () => { 
    setIsEmpty(pw === '' || pwch === '');
  };
```

Reply:
이런 생각을 못했습니다..
하나 배워갑니다!